package.json

```diff
@@ -44,6 +44,10 @@
     "firebase-deploy:staging": "firebase deploy --project=staging --only functions:CI,firestore:rules",
     "firebase-deploy:master": "firebase deploy --project=production --config=firebase-production.json --only functions:CI,firestore:rules",
     "firebase-deploy-backup-function": "firebase deploy --project=production --only functions:scheduledFirestoreBackup",
+    "firebase-deploy-deepdao-function:default": "firebase deploy --project=default --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:development": "firebase deploy --project=development --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:staging": "firebase deploy --project=staging --only functions:scheduledDeepDaoOrganizationListUpdate",
+    "firebase-deploy-deepdao-function:production": "firebase deploy --project=production --only functions:scheduledDeepDaoOrganizationListUpdate",
```

Comment:
Just a heads up, if https://github.com/PrimeDAO/prime-deals-dapp/pull/1162 gets merged
you will have to add `--config=firebase-production.json` to the production related commends
and `--config=firebase-testing.json` to non production commends